packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -164,8 +219,8 @@ export const useResizeHandle = (params: UseResizeHandleParams) => {
     (node) => {
       elementRef.current = node;
       if (elementRef.current) {
-        if (currentValue.current === UNMEASURED) {
-          currentValue.current = elementDimension(node, growDirection);
+        if (currentValue.current === UNMEASURED && relative) {
```

Comment:
let's remove the condition nesting
```ts
      elementRef.current = node;
      if (
        elementRef.current &&
        currentValue.current === UNMEASURED &&
        relative
      ) {
        currentValue.current = 0;
        updateElementsAttrs();
      }
```

Reply:
your code only calls `updateElementsAttrs()` when all of the conditions are met, in my code we always call it as soon as we receive the element. This is by design and this code change would break the logic.

Reply:
Sorry typo - I just wanted to flatten the condition. I didn't mean to move the function call inside